pom.xml

```diff
@@ -114,6 +116,18 @@
 
     <build>
         <plugins>
+            <!-- Update they Keyword Whitelist in the Grammar's production RelObjectNameWithoutValue() -->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
```

Comment:
So how does this actually is processed? I am not sure how this exec-maven-plugin is activated in the maven build process?

Reply:
got it from the other PR